Dockerfile

```diff
@@ -89,25 +89,23 @@ RUN apt-get install -y nodejs
 RUN npm install -g svgo
 
 # Installing and setting up miniconda
-RUN curl -sSLO https://repo.continuum.io/miniconda/Miniconda3-4.2.12-Linux-x86_64.sh && \
-    bash Miniconda3-4.2.12-Linux-x86_64.sh -b -p /usr/local/miniconda && \
-    rm Miniconda3-4.2.12-Linux-x86_64.sh
+RUN curl -sSLO https://repo.continuum.io/miniconda/Miniconda3-4.3.11-Linux-x86_64.sh && \
+    bash Miniconda3-4.3.11-Linux-x86_64.sh -b -p /usr/local/miniconda && \
+    rm Miniconda3-4.3.11-Linux-x86_64.sh
 
 ENV PATH=/usr/local/miniconda/bin:$PATH \
 	LANG=C.UTF-8 \
     LC_ALL=C.UTF-8
 
 # Installing precomputed python packages
-RUN conda config --add channels intel
-ENV ACCEPT_INTEL_PYTHON_EULA=yes
-RUN conda install -y mkl=2017.0.1 \
-                     numpy=1.11.2 \
+RUN conda install -y mkl=2017.0.1 mkl-service &&  \
+    conda install -y numpy=1.12.0 \
                      scipy=0.18.1 \
-                     scikit-learn=0.17.1 \
-                     matplotlib=1.5.3 \
-                     pandas=0.19.0 \
+                     scikit-learn=0.18.1 \
+                     matplotlib=2.0.0 \
```

Comment:
Main effect here seems to be the white matter boundary for EPI registration is now lime green.